internal/requests/device_code_response.go

```diff
@@ -4,40 +4,46 @@
 package requests
 
 import (
-	"encoding/json"
 	"time"
 
+	"github.com/AzureAD/microsoft-authentication-library-for-go/internal/json"
 	"github.com/AzureAD/microsoft-authentication-library-for-go/internal/msalbase"
 )
 
 //DeviceCodeResponse represents the HTTP response received from the device code endpoint
 type DeviceCodeResponse struct {
-	BaseResponse    *msalbase.OAuthResponseBase
+	// TODO(jdoak): Ask someone about why BaseResponse doesn't have a tag.
+	// Either it should be encoded and we should tag it or we should tag it
+	// to be omitted on export or private.
+	BaseResponse    msalbase.OAuthResponseBase
```

Comment:
Not sure, but this almost looks like it was suppose to be used as an embedded type...